### ISSUE TITLE
Add Miami-Dade boundary set to datamade's boundary sets.

### ISSUE THREAD

**Issue by adamold** [Code-for-Miami/mia-councilmatic#29]
Not sure the process for this, but we are currently using Chicago's boundary set. Don't know what all is affected by it, but probably something. 

http://ocd.datamade.us/boundary-sets/

**Comment by ErnieAtLYD** [comment]
Yep - I submitted an issue to the scraper here: https://github.com/opencivicdata/scrapers-us-municipal/issues/116
